roles/container-engine/docker/handlers/main.yml

```diff
@@ -22,14 +22,9 @@
     name: docker
     state: restarted
 
-- name: Docker | pause while Docker restarts
```

Comment:
At the beginning of that same file that is a `notify: Docker | pause while Docker restarts`